python/lbann/models/resnet.py

```diff
@@ -14,7 +14,7 @@ class ConvBNRelu(lbann.modules.Module):
 
     def __init__(self, out_channels, kernel_size, stride, padding,
                  bn_zero_init, bn_statistics_group_size,
-                 relu, name):
+                 relu, name, transpose=False):
```

Comment:
Does ResNet work with deconvolution layers? Won't the outputs become huge?

Reply:
It is not really about ResNet but reusing the Conv-BN-activation pattern. In any case, I will take it out since my immediate need require tinkering with the activation part.